mkdocs.yml

```diff
@@ -144,3 +120,35 @@ extra_css:
 
 watch:
   - includes
+
+nav:
+  - Welcome:
+      - index.md
+      - usage-contrib.md
+      - compatibility.md
+      - tags.md
+  - First Steps:
+      - Requirements: first-steps/requirements.md
+      - Setup: first-steps/setup.md
+      - 3D City Database: first-steps/3dcitydb.md
+      - CityDB tool: first-steps/citydb-tool.md
+      - Docker: first-steps/docker.md
+  - 3D City Database:
+      - Relational Database Schema: 3dcitydb/Relational database schema.md
+      - docker: 3dcitydb/docker.md
+  - CityDB tool:
+      - Database connection: citydb-tool/db-connection.md
+      - Import: citydb-tool/import.md
+      - Export:
+          - Shared Export Options: citydb-tool/export_shared_options.md
+          - CityGML Export: citydb-tool/export_citygml.md
+          - CityJson Export: citydb-tool/export_cityjson.md
+      - Delete: citydb-tool/delete.md
+      - Indexing: citydb-tool/indexing.md
+  - Partners:
+      - Contributors: Partners/contributors.md
+      - Chair od Geoinformatics, TUM: Partners/tum-gis.md
+      - HFT Stuttgart: Partners/hft-stuttgart.md
+      - LIST Eco GmbH & Co. KG: Partners/list-eco.md
+      - M.O.S.S. Computer Grafik Systeme GmbH: Partners/moss.md
+      - Virtual City Systems: Partners/vcs.md
```

Comment:
This is duplicated of what we have per folder in `pages` files.

Reply:
The subsections in the menu on the left are already implemented on `main` already. So this is obsolete anyway.

In general, we have two design options:

@clausnagel @thomashkolbe @marina-rug 
Which one do you like better? 
I prefer option b), as the entire menu remains visible and the does not manually have to expand the subsections.

| a | b |
| - | -| 
|Menu with `>` for subsections|Menu with spacing and grey subheading|
| ![grafik](https://github.com/user-attachments/assets/ff4ac942-cdfc-4a79-a357-5207db164043)  | ![grafik](https://github.com/user-attachments/assets/70914f62-805d-4c7d-bb8b-5fbaac57f003)

Reply:
I think version a) is clearer, because subsections are shown indented. If indentation could be added to b) then this would be fine, too.

Reply:
I fully agree with Thomas :-)

Reply:
I'm fine with option a) too. 
It's implemented on `main` already. See this: https://3dcitydb.github.io/3dcitydb-mkdocs/citydb-tool/